tasks.py

```diff
@@ -974,7 +1041,7 @@ def __create_or_update(ctx, op):
                 getenv("ZOOM_API_KEY"),
                 getenv("ZOOM_API_SECRET"),
                 zoom_admin_id(),
-                "https://{}".format(oc_admin_host),
+                "http://{}".format(oc_admin_host),
```

Comment:
Why is this not HTTPS?

Reply:
Because we've never reworked the opencast admin node to use https/ssl. :(
I didn't even think about it when I first typed "https" there, and then was puzzled when it was causing an error. Then I remembered...

We protect it in all sorts of other ways, but it's really bad that we haven't done this basic step.